vimeo-networking/src/main/java/com/vimeo/networking/model/Connection.java

```diff
@@ -22,16 +22,32 @@
 
 package com.vimeo.networking.model;
 
+import com.google.gson.annotations.SerializedName;
+
+import org.jetbrains.annotations.Nullable;
+
 import java.io.Serializable;
 import java.util.ArrayList;
 
 /**
+ * This model object represents a Connection.
  * Created by hanssena on 4/23/15.
  */
 public class Connection implements Serializable {
 
     private static final long serialVersionUID = -840088720891343176L;
+    @Nullable
+    @SerializedName("uri")
     public String uri;
+    @Nullable
+    @SerializedName("options")
     public ArrayList<String> options;
+    @SerializedName("total")
     public int total;
+    @SerializedName("main_total")
+    public int main_total;
+    @SerializedName("extra_total")
+    public int extra_total;
+    @SerializedName("viewable_total")
+    public int viewable_total;
```

Comment:
Nice addition with the serialized name. I would stay consistent with member naming though with other classes, and use camelCase here, such as ```viewableTotal```. Plus, our GsonBuilder is set up so that if the SerializedName annotation was left off, it would correctly parse ```viewableTotal```.